src/SmartComponents/BaselinesPage/EditBaselinePage/EditBaseline/EditBaseline.js

```diff
@@ -39,13 +39,13 @@ class EditBaseline extends Component {
         this.fetchBaselineId();
     }
 
-    renderHeaderRow(hasWritePermissions) {
+    renderHeaderRow(baselinesWrite) {
```

Comment:
Not sure if this rename was necessary :slightly_smiling_face: .

Reply:
I renamed all of the permission names for consistency. This was done in multiple places.